src/db.rs

```diff
@@ -0,0 +1,14 @@
+use rocksdb::{IteratorMode, DB};
+use serde::{Deserialize, Serialize};
+use std::time::{Duration, SystemTime};
+
+pub const NoAction: u32 = 0b00000000;
+pub const PullRequestCoreDevAuthorIssueNotAssigned24h: u32 = 0b00000010;
+pub const PullRequestCoreDevAuthorIssueNotAssigned72h: u32 = 0b00000100;
```

Comment:
There should be a comment that explains what these are. Are they bit flags?

Reply:
Yes, added comments